HW02_B_Mimic.md

```diff
@@ -49,6 +60,16 @@ Using the iris dataset, make this graph:
 
 ![](HW02_B_Mimic_files/figure-gfm/unnamed-chunk-4-1.png)<!-- -->
 
+```{r graph 2 code, echo=FALSE}
+ggplot(iris,aes(Sepal.Length, Petal.Length, shape = Species, color = Species)) +
+  geom_point() + 
+  geom_smooth(se = FALSE, method = "lm", color = "black") +
+  facet_wrap(.~Species, scales = "free_y") +
+  theme(panel.background = element_rect(linetype = "solid", color = "black"))
+
+#My problem with this is I am not sure how to make the lines appear smoother like in the html example given by the instructor and I have yet to figure out how to swap the order of setosa and versicolor panels.

```

Comment:
I found this to be rather challenging as well.  The most concise way I found was to create a new variable to store the new order in which you want to arrange the panels and then call that variable in facet_wrap where you call ~.Species now (I have added my code at the end of this comment for easy reference.  

For creating a smoother line I get what you are saying.  When I open your figure here the line appears less smooth but when I copy / paste / run your code in my version of RStudio the line is perfectly smooth!  When Robert's figure comes up on my computer I see an image that is not a completely smooth line (it looks more like yours appears) so it might be something with how the image / pixels appear on different computers / screen?


iris$NewOrder <- factor(iris$Species, levels = c("versicolor","setosa","virginica"))

ggplot(iris, aes(Sepal.Length, Petal.Length, color = Species, shape = Species)) +
  geom_point() +
  geom_smooth(method = lm, se = FALSE, color = "black") +
  facet_wrap(vars(NewOrder), scales = "free_y")